core/templates/dev/head/domain/dashboard/DashboardBackendApiService.js

```diff
@@ -33,7 +33,26 @@ oppia.factory('DashboardBackendApiService', [
     });
   };
 
+  var _fetchExplorationStats = function(explorationId) {
+    var requestUrl = (
+      '/dashboardhandler/explorationstats/?exp_id=' + explorationId);
+    return $q(function(resolve, reject) {
+      $http.get(requestUrl, {
+        cache: true
+      }).then(function(response) {
+        if (resolve) {
```

Comment:
To be clearer, you should be able to do this:

    http.get(url).then(function(response) {
      return response.data;
    }).then(function(data) {
      console.log(data) // equal to response.data
    })